ui/component/or-attribute-input/src/scheduler-renderer.ts

```diff
@@ -0,0 +1,101 @@
+import {
+    RankedTester,
+    rankWith,
+    ControlProps,
+    mapStateToControlProps,
+    mapDispatchToControlProps,
+    and,
+    uiTypeIs,
+    formatIs
+} from "@jsonforms/core";
```

Comment:
I would think, if it's necessary to configure some jsonforms specific logic at the "consumer level", then our own `@openremote/or-jsonforms` needs to expose these variables, right?

As in, you don't want `or-attribute-input` to have a dependency on `@jsonforms/core`.

Or, if the `scheduler-render` class is something **every** consumer needs,
then I'd export a `SchedulerRenderer` base class that everyone can extend upon.